cmd/release.go

```diff
@@ -232,6 +232,7 @@ func readChangelog() io.ReadCloser {
 }
 
 // getChangelog returns the changelog's header and body for a given version.
+// Returns empty strings if the given version is not found.
```

Comment:
Why not return just `error` as it is a pattern everywhere?

I would return that error you did in line 127: `errors.New("unable to locate release information in changelog for selected version '" + version + "'."))`
and to fatal in case of error in caller side (:

Reply:
Makes sense, I initially tried to keep the return values unchanged to avoid other changes.